src/templates/pycontw-2020/contents/zh/about/privacy_policy.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'contents/_base.html' %}
+
+{% block title %}PyCon Taiwan 個人資料保護聲明{% endblock title %}
+
+{% block content %}
+
+<section class="pill-title alt">
+	<h1>PyCon Taiwan 個人資料保護聲明</h1>
+</section>
+<h5 style="text-align:right;">(CC BY-SA 3.0 TW)<br/>最後更新時間：2020-08-04</h5>
+<h4>個人資料保護政策</h4>
+
+<p>感謝您參與臺灣 Python 年會 (以下簡稱「PyCon TW」)。「PyCon TW 主辦團隊」(以下簡稱「我們」)
+	將依個人資料保護法之規定來使用、保管您於報名時所提供的個人資料，並維護您的隱私權。同時，我們在此對於您個人資料之使用蒐集將依下列聲明為利用、管理。</p>
+
+<h4>個人資料之使用目的</h4>
+
+<p>我們為求本活動順利進行及提供您相關之服務，蒐集並於使用目的範圍內利用您所提供之個人資料，<b>其法定特定目的為一三〇會議管理</b>。使用目的包含但不限於：</p>
+<ol>
+	　
+	<li>確認報名者與前來報到者之身分同一性</li>
+	　
+	<li>提供您的名牌</li>
+	　
+	<li>其他有助於活動順利進行之相關目的</li>
+</ol>
```

Comment:
excessive line break
![image](https://user-images.githubusercontent.com/24987826/89688987-f24f5180-d935-11ea-9b3d-99a19b4110e6.png)

```suggestion
<ol>
	<li>確認報名者與前來報到者之身分同一性</li>
	<li>提供您的名牌</li>
	<li>其他有助於活動順利進行之相關目的</li>
</ol>
```